elliott/elliottlib/cli/shipment_cli.py

```diff
@@ -44,15 +45,23 @@ def __init__(
         advisory_key: str,
     ):
         self.runtime = runtime
-        self.application = application
-        self.snapshot = snapshot
+        self.application = application or KonfluxImageBuilder.get_application_name(self.runtime.group)
```

Comment:
there are no dots in the application name.
i.e we have to do:
```suggestion
        self.application = application or KonfluxImageBuilder.get_application_name(self.runtime.group).replace(".","-")
```